serialize/internal.thrift

```diff
@@ -0,0 +1,14 @@
+struct RPC {
+	1: required binary spanContext
```

Comment:
> This presumably carries both baggage and tracing.

It does. But we have never used it with Jaeger clients, I think it's not implemented in most languages, so would need testing & extra dev. Why not use `map<string,string>`?